app/views/wrangling_guidelines/_wrangling_guideline_form.html.erb

```diff
@@ -1,42 +1,42 @@
-<%= form_for(@wrangling_guideline, html: { class: 'post' }) do |f| %>
+<%= form_for(@wrangling_guideline, html: { class: "post" }) do |f| %>
 
   <%= error_messages_for @wrangling_guideline %>
 
-  <p class="required notice"><%= ts('* Required information') %></p>
+  <p class="required notice"><%= t(".required_information") %></p>
   <dl>
 
-    <dt class="required"><%= f.label :title, ts('Title') + '*' %></dt>
+    <dt class="required"><%= f.label :title, t(".title") %></dt>
     <dd>
       <%= f.text_field :title %>
-      <%= live_validation_for_field('wrangling_guideline_title', :failureMessage => ts('Please enter a title.')) %>
+      <%= live_validation_for_field("wrangling_guideline_title", failureMessage: t(".title_failure")) %>
     </dd>
 
-    <dt class="required"><%= f.label :content, ts('Guideline text') + '*' %></dt>
+    <dt class="required"><%= f.label :content, t(".guideline_text"), for: "content" %></dt>
     <dd class="required">
       <ul class="hidden rtf-html-switch actions" role="menu">
-        <li><a class="rtf-link" href="#"><%= ts('Rich Text') %></a></li>
-        <li><a class="html-link" href="#"><%= ts('HTML') %></a></li>
+        <li><a class="rtf-link" href="#"><%= t(".rich_text") %></a></li>
+        <li><a class="html-link" href="#"><%= t(".html") %></a></li>
```

Comment:
I think this may confuse us in the future because `html` is a standard key used to mark translations *containing* HTML in html emails. Can you suffix these two keys about the editor type with `_editor`?